src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write code here
+function convertToNumber(value) {
+  return +value.replace(/[^0-9.]/g, '');
+}
+
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const rows = tbody.querySelectorAll('tr');
+const sort = (clickEvent) => {
```

Comment:
```suggestion
const rows = tbody.querySelectorAll('tr');

const sort = (clickEvent) => {
```
sort - не понятное имя